azkaban-common/src/main/java/azkaban/jobExecutor/ProcessJob.java

```diff
@@ -46,12 +49,16 @@
 
   public static final String COMMAND = "command";
   public static final String AZKABAN_MEMORY_CHECK = "azkaban.memory.check";
-  public static final String NATIVE_LIB_FOLDER = "azkaban.native.lib";
```

Comment:
Good thinking. Do you think it's worth creating a NATIVE_LIB_FOLDER string in Constants then? Or should we just keep the strings that are local to ProcessJob and HadoopSecurityManager_H_2_0?